index.js

```diff
@@ -9,8 +9,9 @@ const FastBootExpressMiddleware = require('fastboot-express-middleware');
 const FastBoot = require('fastboot');
 const chalk = require('chalk');
 
-const fastbootAppModule = require('./lib/utilities/fastboot-app-module');
+const fastbootAppBoot = require('./lib/utilities/fastboot-app-boot');
 const FastBootConfig = require('./lib/broccoli/fastboot-config');
+const FastBootAppFactory = require('./lib/broccoli/fastboot-app-factory-plugin');
```

Comment:
out of curiosity, what is the benefit to have such broccoli plugin over [broccoli-file-creator](https://github.com/rwjblue/broccoli-file-creator)?

Reply:
> out of curiosity, would it make sense/be possible to use [broccoli-file-creator](https://github.com/rwjblue/broccoli-file-creator) over new broccoli plugin?

Thank you for your suggestion. I wasn't aware of that broccoli plugin. I gave a glance and looks like I can use this plugin instead of creating a new one. I will give a try.

Reply:
I made changes to use ` broccoli-file-creator` plugin instead of creating a new plugin.